### DISCUSSION TITLE
How do I adjust the Joblist to recognize/respect the filename structure of import tracks?

### DISCUSSION THREAD

**Discussion by Saijin-Naib** [enzo1982/freac#437] [Help & Support]
I've been setting up all my ripping programs for years to use the following format:  
```<track> <title> - <artist> [<album> - <year>].extension```

When importing ripped WAVs into the Joblist of FRE:AC, it does not recognize and segment the filenames into the proper metadata.

How can I tune this?

Thanks!

**Comment by cleanyoursockets** [comment]
Under general settings > playlists > output filenames you have the possibility to change the settings to your liking.

**Comment by cleanyoursockets** [comment]
![Screenshot_2023-02-10_21-13-17](https://user-images.githubusercontent.com/95030469/218189010-96d24ad9-1051-4036-84b5-e5ec04081f7b.png)

**Reply by Saijin-Naib** [reply]
Ah, I think the missing piece here is that these are untagged WAV files with no metadata that I want to load into FRE:AC to be converted to OPUS. I was hoping that there was a way to map the filename into the various metadata fields when being loaded.

It imports like this:  
![image](https://user-images.githubusercontent.com/19295950/218241176-57686978-e8c9-48b9-aab2-039e22e6e61d.png)

And the files are named like this:  
![image](https://user-images.githubusercontent.com/19295950/218241205-f63aeaf8-47bb-460e-8f8f-0bb6ee73aba7.png)

**Reply by enzo1982** [reply]
Unfortunately, the file name structures recognized when no metadata is present are not configurable in fre:ac yet.

However, new [continuous builds](https://github.com/enzo1982/freac/releases/continuous) of fre:ac allow turning metadata recognition from file names off on the _Metadata->Tag fields_ page in the configuration dialog. So you can at least prevent bad metadata recognition.

**Reply by Saijin-Naib** [reply]
Awesome, thanks so much for explaining! This was something I guess I relied upon too much with MP3Tag on Windows to sort of "seed" my files with metadata before I'd go back in and cleanup and add missing bits.
